sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala

```diff
@@ -260,6 +260,20 @@ case class Join(
     condition: Option[Expression])
   extends BinaryNode with PredicateHelper {
 
+  private def isBrocastHint(plan: LogicalPlan): Boolean = {
```

Comment:
OK. I've made it as variables to set & copy from other Join operator. Please take a look if this update is good. Thanks.